genlib/comm.h

```diff
@@ -179,10 +179,11 @@ inline void ICommunicator::CommPostMessage(int m, int x) const {
 }
 
 // a helpful little function...
+// TODO: All time handling in the application uses time_t and stores milliseconds in it, though time_t

```

Comment:
If you add a comment here, might also describe what the function actually does, as it's totally non-obvious.